easybuild/easyconfigs/h/hunspell/hunspell-1.7.0-intel-2018b.eb

```diff
@@ -0,0 +1,28 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'hunspell'
+version = '1.7.0'
+
+homepage = 'http://hunspell.github.io/'
+description = """Hunspell is a spell checker and morphological analyzer library and program designed for languages
+ with rich morphology and complex word compounding or character encoding."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = ['https://github.com/hunspell/hunspell/archive/']
+sources = ['v%(version)s.tar.gz']
```

Comment:
checksum missing

Reply:
Ups! Sorry. Added them and it was missing in the original build file as well.